georaster-layer-for-leaflet.js

```diff
@@ -442,6 +445,57 @@ const GeoRasterLayer = L.GridLayer.extend({
     }
   },
 
+  /**
+   * The callback used to determine the colour based on the values of each pixel
+   *
+   * @callback pixelValuesToColorFn
+   * @param {array | number} values - The pixel value depends on the image being rendered. Multiband images will be a tuple of one value per band. Single band images will be a single number
+   * @returns {string} - Any valid CSS color string
+   */
+
+  /**
+   * Redraws the active map tiles updating the pixel values using the supplie callback
+   * @param {pixelValuesToColorFn} pixelValuesToColorFn - Callback that handles getting the pixel color
+   * @param {Object} [options] - Configuration options passed to the method
+   * @param {boolean} [options.debugLevel=0] - Overrides the global `debugLevel`. Set it to >=1 to allow output here when the global `debugLevel` = 0
+   */
+  updateColors(pixelValuesToColorFn, { debugLevel = 0 } = {}) {
+    if (!pixelValuesToColorFn) {
+      throw new Error("Missing pixelValuesToColorFn function");
+    }
+
+    if (debugLevel >= 1 || this.debugLevel >= 1) console.log("Start updating active tile pixel values");
```

Comment:
I believe this will still log when this.debugLevel is 1 and debugLevel is 0.  Could we do 
```
    if (debugLevel >= 1) console.log("Start updating active tile pixel values");
```

Reply:
True.